### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by cloudhead**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,3 +124,36 @@ TXT record, usually labeled "Seed API" to your HTTP API endpoint, eg.
...
+    $ docker build -t radicle-services/http-api -f http-api/Dockerfile .
+
+#### Running
+
+To run the org node after it is built, you can run for example:
+
+    $ docker run radicle-services/org-node \
+        -v $HOME/.radicle:/app/radicle -- \
+        --subgraph https://api.thegraph.com/subgraphs/name/radicle-dev/radicle-orgs \
+        --orgs 0xceAa01bd5A428d2910C82BBEfE1Bc7a8Cc6207D9
```
Comment (adaszko) [inline README.md:146]: The `-v` needs to go *before* the image name, so:

```
docker run -v $HOME/.radicle:/app/radicle \
        radicle-services/org-node \
        --subgraph https://api.thegraph.com/subgraphs/name/radicle-dev/radicle-orgs \
        --orgs 0xceAa01bd5A428d2910C82BBEfE1Bc7a8Cc6207D9
```

File: README.md [hunk 2]
```diff
@@ -124,3 +124,36 @@ TXT record, usually labeled "Seed API" to your HTTP API endpoint, eg.
...
+Make sure the *identity* file can be found under `$HOME/.radicle/identity` and
+that you replace the org address with your own. Don't forget to specify the
+`--bootstrap` option if needed.
+
+To run the HTTP API, run:
+
+    $ docker run radicle-services/http-api \
+        -v $HOME/.radicle:/app/radicle -- \
+        --tls-cert /app/radicle/fullchain.pem \
+        --tls-key /app/radicle/privkey.pem
```
Comment (adaszko) [inline README.md:157]: Same as above:

```
docker run -v $HOME/.radicle:/app/radicle radicle-services/http-api --tls-cert /app/radicle/fullchain.pem --tls-key /app/radicle/privkey.pem
```

Also, it fails for me with
```
Required options not provided:
    --root
```

File: README.md (hunk 1 above)
Comment (cloudhead) [reply README.md:146]: Aha!

File: README.md (hunk 2 above)
Comment (cloudhead) [reply README.md:157]: Strange, is the CMD not being used?

**Comment by sebastinez** [comment]
**Yeah Docker! :tada:** 
@cloudhead It seems I have an issue with building the org-node Docker image on Ubuntu 20.04 I get the following error, I was able to fix it by using `rustlang/rust:nightly` in the build stage instead of `rustlang/rust:nightly-slim``
```
error: failed to run custom build command for `openssl-sys v0.9.65`

Caused by:
  process didn't exit successfully: `/usr/src/radicle-client-services/target/release/build/openssl-sys-bf67240d15af5f88/build-script-main` (exit status: 101)
  --- stdout
  cargo:rustc-cfg=const_fn
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_OPENSSL_LIB_DIR
  X86_64_UNKNOWN_LINUX_GNU_OPENSSL_LIB_DIR unset
  cargo:rerun-if-env-changed=OPENSSL_LIB_DIR
  OPENSSL_LIB_DIR unset
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_OPENSSL_INCLUDE_DIR
  X86_64_UNKNOWN_LINUX_GNU_OPENSSL_INCLUDE_DIR unset
  cargo:rerun-if-env-changed=OPENSSL_INCLUDE_DIR
  OPENSSL_INCLUDE_DIR unset
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_OPENSSL_DIR
  X86_64_UNKNOWN_LINUX_GNU_OPENSSL_DIR unset
  cargo:rerun-if-env-changed=OPENSSL_DIR
  OPENSSL_DIR unset
  cargo:rerun-if-env-changed=OPENSSL_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=OPENSSL_STATIC
  cargo:rerun-if-env-changed=OPENSSL_DYNAMIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_STATIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_DYNAMIC
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  run pkg_config fail: "Failed to run `\"pkg-config\" \"--libs\" \"--cflags\" \"openssl\"`: No such file or directory (os error 2)"

  --- stderr
  thread 'main' panicked at '

  Could not find directory of OpenSSL installation, and this `-sys` crate cannot
  proceed without this knowledge. If OpenSSL is installed and this crate had
  trouble finding it,  you can set the `OPENSSL_DIR` environment variable for the
  compilation process.

  Make sure you also have the development packages of openssl installed.
  For example, `libssl-dev` on Ubuntu or `openssl-devel` on Fedora.

  If you're in a situation where you think the directory *should* be found
  automatically, please open a bug at https://github.com/sfackler/rust-openssl
  and include information about your system as well as this message.

  $HOST = x86_64-unknown-linux-gnu
  $TARGET = x86_64-unknown-linux-gnu
  openssl-sys = 0.9.65


  It looks like you're compiling on Linux and also targeting Linux. Currently this
  requires the `pkg-config` utility to find OpenSSL but unfortunately `pkg-config`
  could not be found. If you have OpenSSL installed you can likely fix this by
  installing `pkg-config`.

  ', /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/openssl-sys-0.9.65/build/find_normal.rs:174:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `radicle-org-node v0.1.0 (/usr/src/radicle-client-services/org-node)`, intermediate artifacts can be found at `/usr/src/radicle-client-services/target`
```

File: README.md (hunk 2 above)
Comment (sebastinez) [reply README.md:157]: Hmm I'm having the same issue, it seems that CMD is being overriden when one passes additional arguments

> If the user specifies arguments to docker run then they will override the default specified in CMD.
https://docs.docker.com/engine/reference/builder/#cmd

**Comment by cloudhead** [comment]
Uh that's weird, isn't docker supposed to work the same on all systems? Is there a way we can add ssl/pkg-config while keeping rust-slim?

**Comment by cloudhead** [comment]
@sebastinez can you try again with the recent changes?

**Comment by sebastinez** [comment]
@cloudhead Works great! :clap:

**Review by adaszko — APPROVED** [review]

**Review by sebastinez — APPROVED** [review]

**Pull request merged**